### DISCUSSION TITLE
export to excel?

### DISCUSSION THREAD

**Discussion by muppethero** [audiamus/AaxAudioConverter#78] [General]
Just got done converting my 360 books. Love it! 
You have a nice list of my books that I wish I could copy and paste into excel or export that list into excel.

**Comment by audiamus** [comment]
AAX Audio Converter does not use a database or any other mode of persistent storage for the work done.

However, a future version could perhaps scan the output folder hierarchy and create a CSV list from that.

**Comment by muppethero** [comment]
Can you make the text of the list able to copy to the clipboard?

**Reply by audiamus** [reply]
I would normally write it to file. Could be copied to clipboard as well I guess.
